src/widget/taffy_layout.rs

```diff
@@ -0,0 +1,454 @@
+// Copyright 2022 The Druid Authors.
```

Comment:
The occasional uses of the absolute path to reference `taffy` in this file are very unusual. Is this be referring to a different `taffy` (module vs crate?)? I can't see the module/other import in that case

Reply:
Ah, this is an artefact of a workaround I was using (an inline `mod taffy`) to workaround the fact that Taffy wasn't exporting most types at the root level. That has now been corrected so I've updated everything to import from `taffy::`.